Toshi/Source/Toshi/Core/THPTimer.h

```diff
@@ -8,20 +9,20 @@ namespace Toshi
 
 		uint32_t GetRaw32();
 		int64_t GetRaw64();
+
 		void Reset();
+		void Update();
 
 	public:
 		inline float GetDelta() const { return m_fDelta; }
+		inline float GetCurrentSeconds() const { return m_fCurrentSeconds; }
 
 	private:
 		LARGE_INTEGER m_iFrequency;        // 0x0
-		float m_fCurrent;                  // 0x8
-		LARGE_INTEGER m_iUnk;              // 0x10
-		LARGE_INTEGER m_iUnk2;             // 0x14
```

Comment:
Whats with m_iUnk2? Why isn't it used in Reset() anymore?

Reply:
it doesn't even exist, LARGE_INTEGER's size is 8, not 4
JPOG is compiled in x86 so it doesn't support 8-byte types natively - the compiler splits it into two 4-bytes variables

Reply:
Ooooooh i totally forgot about that